src/pages/components/modal/usage.mdx

```diff
@@ -495,20 +496,20 @@ ways.
 
 - **Task completion**: Clicking the primary action will complete the task and
   automatically close the modal.
-- **Cancel button**: Clicking the Cancel button will close the modal and return
+- **Cancel button**: Clicking the **Cancel** button will close the modal and return
   the user to its previous context. Cancel undoes all applied changes.
-- **x**: Clicking the close `x` icon in the upper right will close the modal
+- **x**: Clicking the close **x** icon in the upper right will close the modal
   without submitting any data and return the user to its previous context.
-- **Esc**: Press `ESC` on the keyboard.
+- **Esc**: Press ESC on the keyboard.
 
 #### Button group
 
-There are three buttons in the progress modal footer: Cancel, Previous, Next.
-Each button’s width is 25% of the dialog window and are full bleed. Previous and
-Next are grouped together and placed on the right half of the dialog, with
-Previous as a secondary button and Next as a primary button. In the last step of
-the sequence, the Next button label should change to reflect the final action.
-The Cancel button is aligned to left side of the dialog and uses a ghost button.
+There are three buttons in the progress modal footer: **Cancel**, **Previous**, **Next**.
```

Comment:
Bolding all these words has added a lot of visual clutter in the paragraph. Is there a different style we could use? Or is the bolding necessary here? 
<img width="907" alt="image" src="https://github.com/user-attachments/assets/0b40ef65-0b93-4d3f-a4de-3716c2dba57a">

Reply:
According to IBM Style, we need to bold UI control names. Because we are referring to specific button labels, I do think that they need to be bolded. 
https://ibmdocs-test.dcs.ibm.com/docs/en/ibm-style?topic=elements-ui#button